api4/drafts.go

```diff
@@ -124,7 +124,6 @@ func deleteDraft(c *Context, w http.ResponseWriter, r *http.Request) {
 		switch {
 		case err.StatusCode == http.StatusNotFound:
 			// If the draft doesn't exist in the server, we don't need to delete.
-			mlog.Debug("Unable to find the draft", mlog.Err(err))
```

Comment:
Btw, @mylonsuren - just a note that we don't want to directly call `mlog` functions here. The idiomatic way is to use the logger that's in the `*Context` and call its methods - `c.Logger.Debug(...)`

Reply:
Thanks @agnivade, noted!